tests/simple/testdata/comparisons.textproto

```diff
@@ -147,12 +147,6 @@ section {
     expr: "'Am\\u00E9lie' == 'Ame\\u0301lie'"
     value: { bool_value: false }
   }
-  test {
-    name: "no_string_normalization_surrogate"
-    description: "Should not replace surrogate pairs."
-    expr: "'\\U0001F436' == '\\xef\\xbf\\xbd\\xef\\xbf\\bd'"
-    value: { bool_value: false }
-  }
```

Comment:
So many things wrong with this test:
- no surrogate pairs in sight
- the byte sequence on the right is two replacement characters, perhaps the result of a failed attempt to convert a surrogate pair to UTF-8
- which makes no sense, because UTF-8 has no surrogate pairs
- meaning the test itself also makes no sense because there's no way a surrogate pair could be preserved in a CEL string
- Also, the byte sequence on the right isn't a byte sequence since string literals interpret byte escapes as code points

It's possible I got some of that wrong, but I am pretty sure it's safe to remove this test.